debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlConnectorIT.java

```diff
@@ -903,6 +904,14 @@ public void shouldProcessCreateUniqueIndex() throws SQLException, InterruptedExc
                 .with(MySqlConnectorConfig.INCLUDE_SCHEMA_CHANGES, true)
                 .build();
 
+        try (MySqlTestConnection db = MySqlTestConnection.forTestDatabase(DATABASE.getDatabaseName());) {
+            try (JdbcConnection connection = db.connect()) {
+                connection.execute(
+                 "create table migration_test (id varchar(20) null,mgb_no varchar(20) null)",
+                 "create unique index migration_test_mgb_no_uindex on migration_test (mgb_no)");
```

Comment:
Rearranged these statements such that the connector doesn't fail on starting. Have done the same for the remaining failing tests.